01.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -1211,6 +1212,115 @@ Use `@@` to negate `$removeheader`:
 
 > **Compatibility with different versions of AdGuard.** Rules with `$removeparam` modifier are supported by AdGuard for Windows, Mac, Android, and AdGuard Browser extension for Chrome, Firefox, Edge.
 
+
+<a id="hls-modifier"></a>
+#### **`$hls`**
```

Comment:
We need a disclaimer that these rules are only supported in the versions of AdGuard that use CoreLibs 1.10 or newer (i.e. nightlies for now)